app/templates/components/email-input.hbs

```diff
@@ -1,27 +1,32 @@
-<div class='row'>
-    <div class='label'>
-        <dt>Email</dt>
-    </div>
-    {{#if isEditing }}
+{{#if isEditing }}
```

Comment:
Good call on moving the check for editing out to be able to separate the markup between editing and not editing :) 💯